File: JavaSE/Day 29/shop/apple/ManageProducts.java
package shop.apple;

import java.util.Scanner;

public class ManageProducts {
    public void displayPMenu() {
        Scanner scanner = new Scanner(System.in);
        ProductServices p = new ProductServices();
        System.out.println("Welcome To Product Manager");

        while (true) {
            System.out.println();
            System.out.println("----------------------");
            System.out.println("|   Product Manager  |");
            System.out.println("----------------------");
            System.out.println("(1) Add Product");
            System.out.println("(2) Delete Product");
            System.out.println("(3) View Product");
            System.out.println("(0) Back");
            System.out.print("Select a number: ");

            int number = scanner.nextInt();


            switch (number) {
                case 1:
                    p.addProduct();
                    break;
                case 2:
                    p.deleteProduct();
                    break;
                case 3:
                    System.out.println("----------------------");
                    System.out.println("|   Apple Inventory   |");
                    System.out.println("----------------------");
                    p.viewProducts();
                    break;
                case 0:
                    return;
                default:
                    System.out.println("Invalid option, Try again.");
            }
        }

    }
}

File: Android/Dart/shop/apple/ManageProducts.dart
import 'dart:io';
import 'ProductServices.dart';

class Manageproducts {
  void showPMemenu() {
    while (true) {
      print("\n");
      print("----------------------------");
      print("|      Product Manager      |");
      print("----------------------------");
      print("[1] Add Product");
      print("[2] Delete Product");
      print("[3] View Products");
      print("[0] Back");
      stdout.write("Select a number: ");

      String? input = stdin.readLineSync();
      int? number = int.tryParse(input!);
      print("\n");

      switch (number) {
        case 1:
          print("Product Services Add Product...");
          print("\n");
          Productservices.addProduct();
          break;
        case 2:
          print("Product Services Delete Product...");
          Productservices.deleteProduct();
          break;
        case 3:
          print("----------------------------");
          print("|     Apple Inventory      |");
          print("----------------------------");
          Productservices.viewProducts();
          break;
        case 0:
          return;
        default:
          print("Invalid option. Try agin");
      }
    }
  }
}
